packages/linux-jwrdegoede/PKGBUILD

```diff
@@ -225,7 +224,6 @@ CONFIG_FB_VIA_DIRECT_PROCFS y
 CONFIG_VBOXGUEST n
 CONFIG_DRM_VBOXVIDEO n
 CONFIG_VBOXSF_FS n
-CONFIG_MTD_PHRAM=m
```

Comment:
@cawilliamson Removing this line breaks booting into live environment.

@joshskidmore Can you review the rest of the changes this file?